ts41/get.d.ts

```diff
@@ -0,0 +1,93 @@
+import {Split} from './utilities';
+import {Integers} from '../source/utilities';
+
+/**
+Like @see Get but receives an array of strings as a path parameter.
+*/
+type GetWithPath<T, Keys extends readonly string[]> =
+  Keys extends []
+  ? T
+  : Keys extends [infer Head, ...infer Tail]
+  ? GetWithPath<PropertyOf<T, Extract<Head, string>>, Extract<Tail, string[]>>
+  : never;
+
+type ToPath<S extends string> = Split<FixPathSquareBrackets<S>, '.'>;
+
+type FixPathSquareBrackets<S extends string> =
+  S extends `${infer T}[${infer U}]${infer V}`
+  ? `${T}.${U}${FixPathSquareBrackets<V>}`
+  : S;
+
+type ConsistsOnlyOf<S extends string, C extends string> =
+  S extends ''
+  ? true
+  : S extends `${C}${infer Tail}`
+  ? ConsistsOnlyOf<Tail, C>
+  : false;
+
+type IsInteger<S extends string> = ConsistsOnlyOf<S, Integers>;
```

Comment:
```suggestion
type IsIntegerString<S extends string> = ConsistsOnlyOf<S, Integers>;
```

?

Reply:
`NumericalString`?

Reply:
`1.1` is a number, but it's not allowed, so I think "numerical" is to general.

Reply:
I removed this type completely. It was only used once so I replaced its usage with `ConsistsOnlyOf<Key, StringDigit>`.